tests/unittests/CoreText/CTLineTests.mm

```diff
@@ -66,7 +66,7 @@
     CFAttributedStringRef string = (__bridge CFAttributedStringRef)getAttributedString();
     CTTypesetterRef ts = CTTypesetterCreateWithAttributedString(string);
     CFRange range = { 0, CFAttributedStringGetLength(string) };
-    CTLineRef line = CTTypesetterCreateLineWithOffset(ts, range, 0.0f);
```

Comment:
We currently don't actually do anything with the offset, but I could at least add a test to verify they produce the same results with 0.0f

---
In reply to: [74523272](https://github.com/Microsoft/WinObjC/pull/873#discussion_r74523272) [](ancestors = 74523272)